site/_js/web-components/web-tabs.js

```diff
@@ -88,6 +88,28 @@ export class WebTabs extends BaseElement {
     });
   }
 
+  _getDefaultTab() {
```

Comment:
Nit: this is not default tab, but the tab selected via url param. I'd call it _getSelectedTabIndex